### DISCUSSION TITLE
2023 Community Survey Results

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#12876] [Announcements]
As promised, here are the results of our [2023 NetBox community survey](https://github.com/netbox-community/netbox/discussions/12392)! We received 487 total responses - coincidentally, the same exact number as last year. (Getting the survey in front of a greater portion of the community continues to be a challenge, as a large portion of end users aren't active on GitHub and Slack.)

While every reasonable effort has been made to ensure the integrity of the survey data, bear in mind that each survey response is entirely self-reported and reflects the perspective of the individual respondent.

### 1. Which best describes your organization's industry?

![q1](https://github.com/netbox-community/netbox/assets/13487278/5ca3735e-3891-4af4-82f9-23aff5223188)

### 2. In what country is your organization based?

![q2](https://github.com/netbox-community/netbox/assets/13487278/290f30ef-b99b-4c33-8475-34dd375a1f01)

![q2b](https://github.com/netbox-community/netbox/assets/13487278/fc2b8c07-6a4f-4e21-8ca3-51df9d239580)

### 3. How large is your organization (head count)?

![q3](https://github.com/netbox-community/netbox/assets/13487278/f0c29a96-d6e4-4937-91b0-47b2aa7fb9fb)

### 4. How many NetBox users does your organization have?

![q4](https://github.com/netbox-community/netbox/assets/13487278/a7669b10-3ac4-41ef-9bfe-80c665d3d53e)

### 5. Which teams outside of the networking group utilize NetBox routinely?

![q5](https://github.com/netbox-community/netbox/assets/13487278/440bb486-b70a-4458-86e9-0fb218f3b7a7)

### 6. Roughly how long have you been using NetBox?

![q6](https://github.com/netbox-community/netbox/assets/13487278/b9d4f0dd-17c9-4331-81d4-dcb73da80f06)

### 7. How prevalent is automation in your network?

![q7](https://github.com/netbox-community/netbox/assets/13487278/593fe3e1-4583-449e-8a49-7bacc878909c)

### 8. What version of NetBox are you running in production?

![q8](https://github.com/netbox-community/netbox/assets/13487278/50aad934-fa32-4069-b926-56a70389a0f6)

There's a significant selection bias occurring here, as users who are running more recent releases are inherently more likely to be engaged in the community and therefore more likely to participate in the survey. Still, it's encouraging to see such great adoption of newer releases!

### 9. Approximately how many devices do you manage in NetBox?

Respondents reported an average of **2,832 devices** per NetBox instance, with a maximum of 100 thousand devices.

### 10. Approximately how many prefixes do you manage NetBox?

Respondents reported an average of **2,573 prefixes** per NetBox instance, with a maximum of 170 thousand prefixes.

### 11. How do you deploy NetBox in production?

![q11](https://github.com/netbox-community/netbox/assets/13487278/9300939c-b46f-4240-aa20-1a396f0c369f)

### 12. How frequently do you upgrade to a new release? (Best match)

![q12](https://github.com/netbox-community/netbox/assets/13487278/918f2910-de25-412f-950d-69334088ccc1)

### 13. What types of resources do you manage with NetBox?

![q13](https://github.com/netbox-community/netbox/assets/13487278/97acee07-331f-491e-b4fe-b29bd7e70738)

### 14. Which of the following NetBox features do you use?

![q14](https://github.com/netbox-community/netbox/assets/13487278/ae4cca79-5f4c-4e4b-bea0-b2c63e1f43f0)

### 15. In which of the following areas would you be most excited to see plugins developed?

![q15](https://github.com/netbox-community/netbox/assets/13487278/4bad7185-798e-4d76-97b9-0bf9a3640e7f)

### 16. If you could add one feature to NetBox, what would it be?

277 respondents provided a suggestion here, with a very wide range of topics. Some of these we've seen in past surveys, while others are completely new. Unfortunately, many of the responses we received lack sufficient context to understand what is being requested. Still, we'll lean on these responses when evaluating the popularity of specific suggestions moving forward.

Thank you to everyone who participated in this year's survey!

**Comment by llamafilm** [comment]
Does this survey include paid users of Netbox Cloud? I’m curious to see how popular that is compared with the other deployment options.
